did_doc_sov/src/extra_fields/aip1.rs

```diff
@@ -0,0 +1,4 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Default)]
+pub struct ExtraFieldsAIP1 {}
```

Comment:
There's a type in `shared_vcx` that I made when working on the `messages` one (it was in messages initially but then moved to shared_vcx in a good first issue) and it serves the exact purpose of serializing to nothing and deserializing from thin air. 

It's called `SerdeIgnored` or something like that. What do you think about re-using that instead of another type like `ExtraFieldsAIP1`?

Reply:
Interesting! Didn't know about the flattening behavior, I guess I was just lucky in this case 😅 

I have no issue with reusing `SerdeIgnored`, except I am not too fond of pulling in a dependency (`shared_vcx`) just for that, to be honest...

Reply:
Hmmm... I get you, but I thought the whole point of `shared_vcx` was to allow reusing utilities across the other crates, or am I misunderstanding?

If it isn't, maybe it would be worth creating one like that. The `OneOrList` type you made could go in there too.

I guess it shouldn't be a big deal to pull in a local dependency from a compilation standpoint, as it will always have one version across all places used.

Reply:
Just to clarify, this isn't a showstopper so disregard if it's unimportant 😅 . I'm just brainstorming about reusing utils that we make out of necessity.